### PR TITLE
PMM-5231 Wrong keys in rds_exporter config.

### PR DESCRIPTION
**Pull request by percona-csalguero**
The updateRDSExporterConfig function was receiving a service as a
parameter and it was using AWS credentials from that service, but we can
have more than one service so, we need to get credentials for each
service individually.

### CODE REVIEW

**Comment by puneet0191** [comment]
@percona-csalguero I have tested this and shared the test log and finding with you, please ping me on slack for details. The configuration file keys are correctly mapped but we have issues in the rds_exporter log. 
I have tested feature build: https://github.com/Percona-Lab/pmm-submodules/pull/608

File: services/rds/rds.go [hunk 1]
```diff
@@ -502,6 +502,12 @@ func (svc *Service) updateRDSExporterConfig(tx *reform.TX, service *models.RDSSe
 	}
 	for _, n := range nodes {
 		node := n.(*models.RDSNode)
+
+		service := &models.RDSService{}
```
Comment (AlekSi) [inline services/rds/rds.go]:
```suggestion
		var service models.RDSService
```

That's a preferred syntax for zero values in PMM code base. See also https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_use_a_consistent_declaration_style

File: services/rds/rds.go [hunk 2]
```diff
@@ -502,6 +502,12 @@ func (svc *Service) updateRDSExporterConfig(tx *reform.TX, service *models.RDSSe
...
+		if e := tx.SelectOneTo(service, "WHERE node_id = ?", node.ID); e != nil {
```
Comment (AlekSi) [inline services/rds/rds.go]:
```go
tx.FindOneTo(&service, "node_id", node.ID)
```
We can and should avoid SQL strings with MySQL-specific placeholders in simplest cases like that one.

File: services/rds/rds.go [hunk 3]
```diff
@@ -502,6 +502,12 @@ func (svc *Service) updateRDSExporterConfig(tx *reform.TX, service *models.RDSSe
 	}
 	for _, n := range nodes {
 		node := n.(*models.RDSNode)
+
+		var service models.RDSSeree
```
Comment (BupycHuk) [inline services/rds/rds.go]: `RDSSeree`?

Comment (AlekSi) [reply services/rds/rds.go]: @percona-csalguero please build it locally

**Comment by AlekSi** [comment]
Now only qan-agent test fails, and it is unlikely we ever fix it.

**Pull request merged**